termtyper/utils/parser.py

```diff
@@ -114,6 +114,23 @@ def _create_user_config(self) -> None:
 
         self._write_to_file()
 
+    def add_default_config(self, section: str, option: str) -> None:
+        d = {
+                "user": DEFAULTS,
+                "theming": THEMING,
+                "paragraph": PARAPHRASE,
+                "mode": MODE,
+                "speed records word": SPEED_RECORDS_WORDS,
+                "speed records time": SPEED_RECORDS_TIME,
+            }
+        if section in d:
+            if option in d[section]:
+                self.set(section, option, str(d[section][option]))
+            else:
+                raise NoOptionError(option, section)
+        else:
+            raise NoSectionError(section)
+
```

Comment:
There are two things I would like to add here:

- Instead of `add_default_config` just override the `get` method. (This way we can get rid of checking in `Option` class too) 

- Make `d` global as there are two places in the code where it is being used

Side note: there are 2 or 3 places where I am using `get_boolean` funtion...we'll have to change it to `bool(self.get(...))` for now

Reply:
I got it! Overriding the `get` method can solve both problems.